CMakeLists.txt

```diff
@@ -490,41 +481,28 @@ if(QUDA_NVSHMEM AND (${COMP_CAP} LESS "700"))
   message(SEND_ERROR "QUDA_NVSHMEM=ON requires at least QUDA_GPU_ARCH=sm_70")
 endif()
 
+if(QUDA_OPENMP)
+  find_package(OpenMP)
```

Comment:
Since this is quarded by QUDA_OPENMP  should we make it `REQUIRED` ?

Reply:
Done.